src/test/java/com/amazon/opendistroforelasticsearch/security/securityconf/impl/v7/ConfigV7Test.java

```diff
@@ -72,7 +72,7 @@ public ConfigV7Test(boolean omitDefaults) {
     }
 
     @Test
-    public void testKibana() throws Exception {
+    public void testDashboards() throws Exception {
         ConfigV7.Kibana kibana;
```

Comment:
Should the class `ConfigV7.Kibana` be renamed to `ConfigV7.OpenSearchDashboards`? Also the variable name should be renamed.

Reply:
As we discussed, we'll take care of this change in the reset of the migration tasks.